src/cpu/x64/jit_brgemm_inner_product_utils.cpp

```diff
@@ -336,8 +336,22 @@ int jit_brgemm_ip_conf_t::get_adjusted_oc_block() const {
     const bool is_f32_compute = !jbgp.is_bf32
             && everyone_is(f32, jbgp.src_dt, jbgp.wei_dt, jbgp.dst_dt);
     const bool is_avx512 = is_superset(jbgp.isa, avx512_core);
+    const bool is_avx2 = is_superset(jbgp.isa, avx2);
     const bool is_f32_compute_avx512 = is_f32_compute && is_avx512;
-
+    const bool is_f32_compute_avx2 = !is_avx512 && is_avx2 && is_f32_compute;
+    // Heuristic to unify oc block.
+    // batch size maybe affect oc_block, which would increased LLM peak USS memory.
```

Comment:
I would say the motivation itself not clear. Would propose to rephrase:
"These heuristic are required to avoid usage different weight layouts in case of different data shapes. Applicibility is limited to big weights only (like LLM use cases) since minimal memory consumption and time for weights reorder are key optimization points there"